trade-finance-manager-ui/server/helpers/deal-cancellation-enabled.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as ukefCommon from '@ukef/dtfs2-common';
+import { DEAL_SUBMISSION_TYPE } from '@ukef/dtfs2-common';
+import { dealCancellationEnabled } from './deal-cancellation-enabled';
+
+describe('dealCancellationEnabled', () => {
```

Comment:
I'd be interested for you to have a conversation with Abhi about how we want to set up these files as I know this is how we have traditionally mocked it but I think we have a new approach, and it'd be good to get notes on the approach that I can add to a living document on testing around feature flags :)

Reply:
Just a heads up -- I'll review this more once this conversation is had, as I feel that this might change a few things!

Reply:
Have refactored using the new `ff` test file namings as Abhi implemented. I think Abhi will do a call with us all in at some point for the full feature flag low down - agreed would be good do have documentation on the approach, happy to create / contribute to that.